### PR TITLE
feat: micro experiment onboarding less text copy

### PR DESCRIPTION
**Pull request by kopancek**
### Before
Screenshot 2024-03-08 at 17 57 29
Screenshot 2024-03-08 at 17 57 16

### With experiment
Screenshot 2024-03-08 at 17 53 35
Screenshot 2024-03-08 at 17 53 07

## Events

Did you introduce any new tracking events?
Don't forget to update the [Analytics Taxonomy sheet](https://docs.google.com/spreadsheets/d/18Lv7zXges9QfVX5VYL1a-Hyl0e1sQ3sLr0OK8YZWKXI/edit#gid=0)

Log the new/changed events below:

| Type   | event_name  | value |
|--------|-------------|-------|
| Change/New | event name  | extra: { ... } |

### On those affected packages:
- [X] Have you done sanity checks in the webapp?
- [ ] Have you done sanity checks in the extension?
- [ ] Does this not break anything in companion?

### Did you test the modified components media queries?
- [X] MobileL (420px)
- [X] Tablet (656px)
- [X] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

MI-213 #done

### CODE REVIEW

**Review by rebelchris — APPROVED** [review]
Lol at this point we should just make this page webflow 🫠

**Review by omBratteng — APPROVED** [review]

**Comment by omBratteng** [comment]
> Lol at this point we should just make this page webflow 🫠

Shh, don't give them ideas 😅

**Review by sshanzel — APPROVED** [review]
Wow. Literally less text. I thought it was going to be a one sentence in exchange.

File: packages/shared/src/components/auth/AuthOptions.tsx [hunk 1]
```diff
@@ -448,6 +450,7 @@ function AuthOptions({
             targetId={targetId}
             className={className?.onboardingSignup}
             onboardingSignupButtonSize={onboardingSignupButtonSize}
+            onboardingSignupButtonVariant={onboardingSignupButtonVariant}
```
Comment (nensidosari) [inline packages/shared/src/components/auth/AuthOptions.tsx:453]: Would these 2 better as a sing json?

**Review by nensidosari — APPROVED** [review]
Looks good.

One small suggestion, non blocker.

Also just checking if you've verified this has changed in the post page banner as well and looks good (I believe we use the same components).

File: packages/shared/src/components/auth/AuthOptions.tsx (hunk 1 above)
Comment (kopancek) [reply packages/shared/src/components/auth/AuthOptions.tsx:453]: Hmm, makes sense. I'll change it to accept more button props 👍

**Comment by kopancek** [comment]
> Also just checking if you've verified this has changed in the post page banner as well and looks good (I believe we use the same components).

Good point, started a [discussion on slack](https://dailydotdev.slack.com/archives/C06K5KAHM0X/p1710143306168439)

**Review by nensidosari — APPROVED** [review]

**Pull request merged**